content/docs/dvclive/api-reference/live/log_plot.md

```diff
@@ -0,0 +1,150 @@
+# Live.log_plot()
+
+```py
+def log_plot(self, name: str, labels, predictions, **kwargs):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+y_true = [0, 0, 1, 1]
+y_pred = [1, 0, 1, 0]
+y_score = [0.1, 0.4, 0.35, 0.8]]
+live.log_plot("roc", y_true, y_score)
+live.log_plot("confusion_matrix", y_true, y_pred)
+```
+
+## Description
+
+Generates a
+[scikit learn plot](https://scikit-learn.org/stable/visualizations.html) and
+saves the result in `{path}/plots/{name}.json`.
+
+💡 The generated `{path}/plots/{name}.json` can be visualized with `dvc plots`.
+
+### Step updates
+
+⚠️ `Live.log_plot()` can't be currently used in scripts where the step is being
+updated ( with `Live.next_step()` or `Live.set_step()`).
+
+## Supported plots
+
+`name` must be one of the supported plots:
+
+### `calibration`
+
+Generates a
+[calibration curve](https://scikit-learn.org/stable/modules/calibration.html#calibration-curves)
+plot.
+
+Calls
+[sklearn.calibration.calibration_curve](https://scikit-learn.org/stable/modules/generated/sklearn.calibration.calibration_curve.html)
+and stores the outputs at `{path}/plots/calibratrion.json` in a format
```

Comment:
I would replace "outputs" in each of these with "data" or something else more specific.